src/Commands/Backup/ListCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * @file
+ * Contains Pantheon\Terminus\Commands\SSHKey\ListCommand
+ */
+
+
+namespace Pantheon\Terminus\Commands\Backup;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Terminus\Collections\Sites;
+use Terminus\Models\Environment;
+
+class ListCommand extends TerminusCommand
+{
+
+    /**
+     * Contructor that supports injection for unit testing.
+     */
+    public function __construct($sites = null)
+    {
+        parent::__construct();
+        $this->sites = $sites ? $sites : new Sites();
+    }
+
+    /**
+     * Lists the Backups for a given Site and Environment
+     *
+     * @authorized
+     *
+     * @command backup:list
+     * @aliases backups
+     *
+     * @param string $environment Name of the environment to retrieve
+     * @param string $element Element filter code/files/database/db (optional)
```

Comment:
I think the typical format for optional args and values in this case might look like:
`* @param string $type Only show backups of a certain type: [code|files|database|db]`